src/Rings/mpoly-localizations.jl

```diff
@@ -3040,3 +3040,128 @@ end
   return dim(saturated_ideal(I))
 end
 
+################################################################################
+#
+# Minimal generating set
+#
+################################################################################
+
+@doc raw"""
+    minimal_generating_set(I::MPolyLocalizedIdeal)
+
+Given an ideal `I` in the localization of a multivariate polynomial ring
+over a field at a point, return an array containing a minimal set of
+generators of `I`. If `I` is the zero ideal an empty list is returned.
```

Comment:
Minimal with respect to inclusion or cardinality?

Reply:
This is a fixed technical term. It is minimal as in minimal presentation of R/I (exactly like in the graded case), which in turn implies minimal w.r.t. cardinality.

Reply:
It is also strict for localizations at prime ideal (seen als localized ring over a field extension) -- but we do not have functionality for this kind of ring in the backend. So I put a note there stating that it is only available for rational points.